backend/chunker/chunker.go

```diff
@@ -2154,13 +2195,14 @@ type metaSimpleJSON struct {
 // - if file contents can be mistaken as meta object
 // - if consistent hashing is On but wrapped remote can't provide given hash
 //
-func marshalSimpleJSON(ctx context.Context, size int64, nChunks int, md5, sha1 string) ([]byte, error) {
+func marshalSimpleJSON(ctx context.Context, size int64, nChunks int, md5, sha1, xactID string) ([]byte, error) {
 	version := metadataVersion
```

Comment:
As we know, the difference between version 1 and 2 is the transaction id field.
If xactId=="" here, let's write version=1 and make sure the go marshaler omits it.
Please add unit test that checks ability to read and write metadata ver.1.

Reply:
The reason why I ask will become clear later, when we discuss `List` and `Move`.